website/docs/foundations/tokens/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Tokens
+caption: Design tokens are provided as CSS custom properties and used to share and standardize foundation styles.
+description: Design tokens are provided as CSS custom properties and used to share and standardize foundation styles.
```

Comment:
@heatherlarsen Updated this one. Since the title was shortened to just "Tokens" I think referring to it as "design tokens" in the caption and description helps clarity. (Since "tokens" can have different meanings in code.)